src/python/ensembl/io/genomio/genome_metadata/dump.py

```diff
@@ -19,22 +19,28 @@
     "filter_genome_meta",
     "check_assembly_version",
     "check_genebuild_version",
+    "metadata_dump_setup",
 ]
 
+import argparse
 import json
 from typing import Any, Dict, Type
 import logging
 
 from sqlalchemy import select
 from sqlalchemy.orm import Session
+from sqlalchemy.engine import URL
 
 from ensembl.core.models import Meta
+from ensembl.io.genomio.utils.json_utils import get_json
 from ensembl.io.genomio.database import DBConnectionLite
 from ensembl.utils.argparse import ArgumentParser
+from ensembl.utils import StrPath
 from ensembl.utils.logging import init_logging_with_args
 
 
-METADATA_FILTER: Dict[str, Dict[str, Type]] = {
+DEFAULT_FILTER: Dict[str, Dict[str, Type]] = {
+    "database": {"name": str},
```

Comment:
Will this pass schema-based validation https://github.com/Ensembl/ensembl-genomio/blob/main/src/python/ensembl/io/genomio/data/schemas/genome.json ?

Reply:
What's the purpose of this property? What if we'll change schema version or something? Is this a technical property? Should we prefix it with `_` or something?